server/apps/users/views.py

```diff
@@ -106,21 +86,30 @@ class LogoutView(APIView):
     @staticmethod
     def post(request):
         response = Response()
+        blacklist_jwt_token(request.COOKIES.get('jwt'))
         response.delete_cookie('jwt')
+        response.delete_cookie('csrftoken')
         response.data = {
             'message': LOGOUT_SUCCESSFUL
         }
         return response
 
 
-class TodoListView(APIView):
+class TodoView(generics.CreateAPIView, generics.RetrieveUpdateDestroyAPIView):
+    # Create to'do
     @staticmethod
-    def get(request):
-        token = request.COOKIES.get('jwt')
+    def post(request, **kwargs):
+        serializer = TodoSerializer(context={'request': request}, data=request.data)
```

Comment:
Why are you passing the context here? As the serializer is not using it.

Reply:
Refactored it. And now serializer is using the context.